acoustid/future/fpindex/client.py

```
@@ -0,0 +1,396 @@
+        if data is not None:
+            kwargs["data"] = msgspec.msgpack.encode(data)
+            kwargs["headers"]["Content-Type"] = "application/vnd.msgpack"
+
+        try:
+            async with self.session.request(method, url, **kwargs) as response:
+                status = response.status
+
+                if expected_status and status not in expected_status:
+                    body = await response.text()
+                    logger.error(
+                        "Unexpected status %s for %s %s: %s", status, method, url, body
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)